src/components/ItemsList/index.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
+
+import ItemsList from './ItemsList';
+import { removeItem, increaseQuantity, decreaseQuantity, updateQuantity } from '../../actions/cartActions';
+
+function mapStateToProps(state) {
+  return {
+    items: state.cart.items
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    handleRemove: cartItemId => dispatch(removeItem(cartItemId)),
```

Comment:
`handleRemove` is targeting the item, `handleIncrease` is targeting quantity. It is not obvious what these methods mutate, when you call them inside component.